dev/tasks/tasks.yml

```diff
@@ -895,6 +895,13 @@ tasks:
       - r-pkg__bin__macosx__big-sur-arm64__contrib__4.3__arrow_{no_rc_r_version}\.tgz
       - r-pkg__src__contrib__arrow_{no_rc_r_version}\.tar\.gz
 
+{% for which in ["strong", "most"] %}
+  r-recheck-{{which}}:
```

Comment:
Do we want to add `r-recheck-*` to https://github.com/apache/arrow/blob/567f9c596ed8e5b9c4435172442997874a4764e0/dev/tasks/tasks.yml#L164-L166 so that it will be automatically run nightly (or maybe just the more expansive of the two?)

We should also add that to https://github.com/apache/arrow/blob/567f9c596ed8e5b9c4435172442997874a4764e0/dev/tasks/tasks.yml#L94-L96 so that it comes along with the `r` group yeah?

Alternatively, we could call this `test-r-recheck` and it would automagically be in both places

Reply:
Those both sound like good suggestions. Though I wonder if running recheck nightly might be a bit of overkill. It takes about 3 hours of total CI time.

Maybe for now I just change the prefix as you suggest to `test-r-recheck` but we leave it off nightly?

Reply:
> change the prefix as you suggest to test-r-recheck but we leave it off nightly?

That doesn't work as the nightly-tests group uses the test- prefix.
I think we can add it to the r group so it's included in any runs on PRs that are started manually (and before the release). With the current low level of changes in the R package I don't think a nightly run of these long jobs is justified.

Reply:
Ah, k.

I'll add `r-recheck-most` (the more expansive of the two revdep checks) to the r group.

Reply:
Added in https://github.com/apache/arrow/pull/43784/commits/a077cc5b3b121a98d1bf0650415892f67c6d34ce.

Reply:
_nods_ if it's not blocking PRs I think it's fine even if it's long, but happy to only run it with r tests + when called direclty. 

If you want that to happen and you change the names to include `test-` you'll need to use the exception pattern to exclude this one from the regex match like at 

https://github.com/apache/arrow/blob/567f9c596ed8e5b9c4435172442997874a4764e0/dev/tasks/tasks.yml#L184

Reply:
Good point.

Just to wrap this conversation up, so with my change in https://github.com/apache/arrow/commit/a077cc5b3b121a98d1bf0650415892f67c6d34ce I think this job will be callable either with `r-*`, `r-recheck-*` or `r-recheck-{most, strong}` and it will not run nightly.

Reply:
Also sorry my comment there is slightly out of sequence. I wrote it before Jacob's comment but then didn't ever press enter for like 5 hours 🤦

Reply:
No worries at all! Thanks for taking a look.